<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyProject</class>
 <widget class="QMainWindow" name="MyProject">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>564</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MyProject</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{background-color: rgb(20, 35, 39);}

QWidget#centralwidget{background-image: url(:/resources/img/амва.PNG);}
QGroupBox{background-image: url(:/resources/img/амва.PNG);}

QPushButton {
color: rgb(255, 255, 255);
     /*border: 2px solid #008B8B;*/
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #99f0ee, stop: 1 #008B8B);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #008B8B, stop: 1 #99f0ee);
 }

 QPushButton:flat {
     border: none; /* для плоской кнопки границы нет */
 }
 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }


QLabel{color: rgb(20, 35, 39);}

QLineEdit{
border: 1px solid /*rgb(89, 145, 166);*/ rgb(197, 197, 197);
border-radius: 40px;
padding-left: 20px;
padding-right: 20px;
}
QLineEdit:hover{
border: 1px solid rgb(61, 105, 125);
border-radius: 40px;
padding-left: 20px;
padding-right: 20px;
}
QLineEdit:focus{
border: 2px solid rgb(106, 209, 208);
border-radius: 40px;
padding-left: 20px;
padding-right: 20px;
}

QSpinBox{
border: 1px solid #008B8B;
border-radius: 20px;
padding-left: 10px;
padding-right: 10px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>501</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="clickCalc">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>330</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GOST type B (plotter)</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Рассчитать ежемесячный платеж, общую сумму выплат и переплату по кредиту&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Рассчитать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GOST type B (plotter)</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Сумма кредита</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="credit_sum">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>223</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>180</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GOST type B (plotter)</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Сроки выплат</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="credit_terms">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>223</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="percent">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>223</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>201</width>
       <height>58</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GOST type B (plotter)</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Процентная ставка</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>руб.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>мес.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>222</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>GOST type B (plotter)</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Дата начала выплат:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="day">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>270</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
     <property name="value">
      <number>17</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="month">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="year">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>270</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>2000</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>2021</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GOST type B</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;с аннуитетным платежом&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">/*color: rgb(0, 85, 127);*/

color: rgb(33, 149, 149);
</string>
    </property>
    <property name="text">
     <string>Кредитный калькулятор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slotClear">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GOST type B (plotter)</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Очистить все значения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(20, 35, 39);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Введите значения!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="click_table">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GOST type B (plotter)</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Показать таблицу с графиком платежей по кредиту&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>График платежей &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>140</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="payment">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>GOST type B (plotter)</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(20, 35, 39);</string>
       </property>
       <property name="text">
        <string>Общая сумма выплат</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <family>GOST type B (plotter)</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(20, 35, 39);</string>
       </property>
       <property name="text">
        <string>Переплата</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>GOST type B (plotter)</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Ежемесячный платеж</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="pereplata">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
